app/schemas/USER.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+      "userId": {
+        "type": "string"
+      },
+      "f_name": {
+        "type": "string"
+      },
+      "l_name": {
+        "type": "string"
+      },
+      "email": {
+        "type": "string"
+      },
+      "phone_number": {
```

Comment:
probably worthwhile in the markdwon to write the format. EX: `1234567890` vs `(123) 456-7890` vs anything else. My vote would be for `1234567890` an let the frontend shape that into the desired form